Library/Homebrew/language/python.rb

```diff
@@ -139,11 +139,22 @@ def virtualenv_create(venv_root, python = "python", formula = self)
         venv
       end
 
+      def needs_python?(python)
+        return true if build.with?(python)
+        (requirements.to_a | deps).any? { |r| r.name == python && r.required? }
+      end
+
       # Helper method for the common case of installing a Python application.
       # Creates a virtualenv in `libexec`, installs all `resource`s defined
       # on the formula, and then installs the formula.
-      def virtualenv_install_with_resources
-        venv = virtualenv_create(libexec)
+      def virtualenv_install_with_resources(options = {})
```

Comment:
```
virtualenv_install_with_resources :python => "python3"
virtualenv_install_with_resources :python => "python"
```
looks more sane to me than
```
virtualenv_install_with_resources "python3"
virtualenv_install_with_resources "python"
```
or
```
virtualenv_install_with_resources("python3")
virtualenv_install_with_resources("python")
```

```
$ virtualenv --help|grep python
  -p PYTHON_EXE, --python=PYTHON_EXE
                        --python=python2.5 will use the python2.5 interpreter
                        (/usr/local/opt/python/bin/python2.7)
```
So there's a case for calling it `:python_exe` if anyone is concerned about confusion with adding a third `:python` to the code, the other two being
```
depends_on :python
```
and
```
depends_on "numpy" => :python
```
Incidentally, I'm wondering if the `depends_on "numpy" => :python` usage should be entirely deprecated.